doc/proposals/ansible-helm-addapi.md

```diff
@@ -0,0 +1,40 @@
+# Ansible/Helm add api command  Proposal for Operator SDK
+
+> Status: implementable
+> 
+
+- [Background](#background)
+- [Goals](#goals)
+- [Commands](#commands)
+- [Ansible_Changes] (#ansible changes)
+- [Helm_Changes] (#helm changes)
+
+## Background
```

Comment:
Following my suggestion for : https://github.com/operator-framework/operator-sdk/blob/master/doc/proposals/TEMPLATE.md#user-stories-optional
 
**Story**

- I am as an Ansible operator-sdk developer, I'd like to be able to scaffold new API"s by using the command `operator-sdk new Api`
- I am as a Helm operator-sdk developer, I'd like to be able to scaffold new API"s by using the command `operator-sdk new Api`
- I am as a Helm/Ansible operator-sdk developer, I'd like to not use the flag --type when the project is already scaffold. 

**Acceptance Criteria**

- Documentation updated with these steps
- Flags options which are related to the API scaffold and available for operator-sdk new project --type [options] also should be provided in the operator-sdk new api for the types. 
- All required files for the new Kind be a scaffold for an Ansible based-operator when the command `operator-sdk new --api-version=<group>/<version> --kind=<Kind>` is executed for example.
- All required files for the new Kind be scaffolded for a Helm based-operator when the command `operator-sdk new memcached-operator --api-version=<group>/<version> --kind=<Kind> --helm-chart=stable/memcached` is executed for example.
 
PS.: I would split by type, so for example, Ansible would be story 1 and Helm story 2